Code/GraphMol/RGroupDecomposition/RGroupDecomp.h

```diff
@@ -70,6 +70,8 @@ class RDKIT_RGROUPDECOMPOSITION_EXPORT RGroupDecomposition {
   RGroupDecomposition(const std::vector<ROMOL_SPTR> &cores,
                       const RGroupDecompositionParameters &params =
                           RGroupDecompositionParameters());
+  RGroupDecomposition(const ROMol &core, const std::string &json);
```

Comment:
The same as above (I would rename the second parameter)

Reply:
See for example MolStandartize.cpp